src/cryptonote_core/service_node_list.cpp

```diff
@@ -723,11 +718,16 @@ namespace service_nodes
     // and if we don't already have the maximum
```

Comment:
This comment should be updated or removed, gone stale now with the new min contribution rules.